instrumentation-api-incubator/src/test/java/io/opentelemetry/instrumentation/api/incubator/semconv/db/SqlClientAttributesExtractorTest.java

```diff
@@ -102,35 +147,65 @@ void shouldNotExtractTableIfAttributeIsNotSet() {
     underTest.onStart(attributes, context, request);
 
     // then
-    assertThat(attributes.build())
-        .containsOnly(
-            entry(DbIncubatingAttributes.DB_STATEMENT, "SELECT *"),
-            entry(DbIncubatingAttributes.DB_OPERATION, "SELECT"));
+    if (SemconvStability.emitOldDatabaseSemconv()) {
+      assertThat(attributes.build())
+          .containsOnly(
+              entry(DbIncubatingAttributes.DB_STATEMENT, "SELECT *"),
+              entry(DbIncubatingAttributes.DB_OPERATION, "SELECT"));
+    } else {
+      assertThat(attributes.build())
+          .containsOnly(
+              entry(
+                  SemconvStabilityUtil.getAttributeKey(DbIncubatingAttributes.DB_STATEMENT),
+                  "SELECT *"),
+              entry(
+                  SemconvStabilityUtil.getAttributeKey(DbIncubatingAttributes.DB_OPERATION),
+                  "SELECT"));
+    }
   }
 
   @Test
   void shouldExtractTableToSpecifiedKey() {
     // given
     Map<String, String> request = new HashMap<>();
-    request.put("db.statement", "SELECT * FROM table");
+    if (SemconvStability.emitOldDatabaseSemconv()) {
+      request.put("db.statement", "SELECT * FROM table");
+    } else {
+      request.put("db.query.text", "SELECT * FROM table");
+    }
```

Comment:
do we test also with `database/dup`? if so should this instead be:

```suggestion
    if (SemconvStability.emitOldDatabaseSemconv()) {
      request.put("db.statement", "SELECT * FROM table");
    }
    if (SemconvStability.emitNewDatabaseSemconv()) {
      request.put("db.query.text", "SELECT * FROM table");
    }
```